### PR TITLE
v0.3.0 — IP Pool support + minor improvements

### PR DESCRIPTION
**Pull request by eexit**
Hello,

* Added IP pool support + test
* Added comments
* Normalized PHP files with [PHP-FIG  recommendations](http://www.php-fig.org/) + set a coding style consistency
* Minor improvements

Review will be easier when ignoring whitespace changes: https://github.com/sendgrid/smtpapi-php/pull/5/files?w=1

Would be really nice to have this merged asap.

Thank you very much,

### CODE REVIEW

File: scripts/s3upload.sh [hunk 1]
```diff
@@ -32,6 +20,18 @@ then
     -F "Content-Type=application/zip" \
     -F "file=@./smtpapi-php.zip" \
     https://s3.amazonaws.com/$S3_BUCKET
+
+if [ "$TRAVIS_BRANCH" = "master" ]
+then
+    curl -X POST \
+        -F "key=smtpapi-php/smtpapi-php.zip" \
```
Comment (mbernier) [inline scripts/s3upload.sh]: Should this have $GIT_VERSION ?

File: test/Smtpapi/Header.php [hunk 2]
```diff
@@ -1,191 +1,219 @@
...
-    $this->assertEquals($this->t['set_filters'], $header->jsonString());
-  }
-
+class SmtpapiTest_Header extends \PHPUnit_Framework_TestCase
+{
+    protected $t;
+
+    protected function setUp()
+    {
+        $file = __DIR__ . '/../smtpapi_test_strings.json';
```
Comment (mbernier) [inline test/Smtpapi/Header.php]: Note for reviewers: in PHP 
`__DIR__` is equivalent to `dirname(__FILE__)`

**Comment by mbernier** [comment]
Other than the question about `$GIT_VERSION`, the changes look legit.

Left  to do: 

- [ ] Confirm `$GIT_VERSION` question
- [ ] Test the changes

File: scripts/s3upload.sh (hunk 1 above)
Comment (eexit) [inline scripts/s3upload.sh]: I have no idea, I didn't write this, I just changed the indentation :)

**Comment by eexit** [comment]
@mbernier I tested this PR in our system integration and it works perfectly. What else test would you like to be done?

**Comment by mbernier** [comment]
We'll run it through our internal tests as soon as we can and then hopefully get it merged.

**Comment by eexit** [comment]
Oh, right! Great!

Thanks :)

**Comment by eddiezane** [comment]
Thanks so much for the PR!

What did you use to lint the codebase for FIG standards?

**Comment by eexit** [comment]
@eddiezane Thanks!
I quickly did it by hand... However, there are some PHP beautifuler online but I never tested yet :)

**Pull request merged**